README.md

```diff
@@ -1,13 +1,11 @@
 ![AWS Quick Start for Kubernets](images/banner.jpg)
 
-# VMware AWS Quickstart
+# DEPRECATED: This project is no longer maintained.
 
-[<img
```

Comment:
FWIW this is specifying that it was certified for k8s 1.13 which hasnt changed (even if deprecated). The `alt` text is wrong though (says 1.11).

Since it is deprecated I dont have a strong preference though.

Reply:
While it _was_ certified for 1.13, per CNCF rules when you fail to maintain conformance, all of your certifications are revoked.

Reply:
👍 Thanks; I wasn't aware of that rule. TIL.